packages/arb-token-bridge-ui/src/components/MainContent/MainContent.tsx

```diff
@@ -27,16 +29,33 @@ export const motionDivProps = {
 }
 
 export function MainContent() {
-  const { closeTransactionHistoryPanel } = useAppContextActions()
   const { address } = useAccount()
-  const transactionHistoryProps = useTransactionHistory(address)
+  const { closeTransactionHistoryPanel } = useAppContextActions()
+  const transactionHistoryProps = useTransactionHistory(address, true)
```

Comment:
```suggestion
  const transactionHistoryProps = useTransactionHistory(address, { runFetcher: true })
```

Reply:
Leave a comment for this to be fixed in the future

Reply:
updated